server/util/redact/redact_test.go

```diff
@@ -282,6 +301,7 @@ func TestRedactMetadata_BuildMetadata_StripsURLSecrets(t *testing.T) {
 	})
 
 	assert.Equal(t, "https://github.com/buildbuddy-io/metadata_repo_url", buildMetadata.Metadata["REPO_URL"])
+	assert.Equal(t, `["--remote_header=\u003cREDACTED\u003e","--foo=SAFE"]`, buildMetadata.Metadata["EXPLICIT_COMMAND_LINE"])
```

Comment:
I think you can use `<` and `>` since they are plain ascii chars?

```suggestion
	assert.Equal(t, `["--remote_header=<REDACTED>","--foo=SAFE"]`, buildMetadata.Metadata["EXPLICIT_COMMAND_LINE"])
```

Reply:
I had this initially, but it seems the JSON marshalling / unmarshalling code converts these into `\u003c` and `\u003e`. They're rendered correctly in the UI so it doesn't seem like a huge deal (there's a way to enable this escaping, but I'd kinda rather leave it enabled because I'm sure there's some reason for it).

What I was really looking for was an assert.Equal that behaves more like `strings.EqualFold()` but couldn't find anything.

Reply:
> I'd kinda rather leave it enabled because I'm sure there's some reason for it

ah I bet it's to support dumping these into HTML without the `<` being treated as a tag or something like that.

Reply:
huh, TIL: https://codepen.io/bduffs/pen/LYXoKoP